.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
+**/node_modules
+dist
+package-lock.json
```

Comment:
You're goning to want to add the wildcards to the package-lock and dist just like the node_modules.